senders/beanstlakd_sender.go

```diff
@@ -15,6 +18,9 @@ import (
 
 var bLogger = log.WithFields(log.Fields{"class": "BeanstalkdSender"})
 
+// Characters allowed in beanstalkd
+const AllowedCharacters = `[A-Za-z0-9\\\-\+\/\;\.\$\_\(\)]+`
```

Comment:
Leave a comment on where you found this regex

Reply:
use lower case for private variables

```suggestion
const allowedCharacters = `[A-Za-z0-9\\\-\+\/\;\.\$\_\(\)]+`
```